.flake8

```diff
@@ -1,19 +1,59 @@
 [flake8]
-# E203 skipped because it annoys black.
-# no pyproject.toml support https://gitlab.com/pycqa/flake8/-/issues/428
-show-source = true
-extend-ignore =
-  E203,
-  F401,
-max-line-length = 100
+
 builtins = _
+
+# Print the total number of errors:
+count = true
+
+# Don't even try to analyze these:
 extend-exclude =
-  # skip tool cache dirs
+  # No need to traverse egg info dir
+  *.egg-info,
+  # tool cache dirs
   *_cache
-  # skip project env vars
+  # project env vars
   .env,
-  # skip adjacent venv
+  # GitHub configs
+  .github,
+  # Cache files of MyPy
+  .mypy_cache,
+  # Cache files of pytest
+  .pytest_cache,
+  # Temp dir of pytest-testmon
+  .tmontmp,
+  # Occasional virtualenv dir
+  .venv
+  # VS Code
+  .vscode,
+  # Temporary build dir
+  build,
+  # This contains sdists and wheels of ansible-navigator that we don't want to check
+  dist,
+  # Metadata of `pip wheel` cmd is autogenerated
+  pip-wheel-metadata,
+  # adjacent venv
   venv
+
+# IMPORTANT: avoid using ignore option, always use extend-ignore instead
+# Completely and unconditionally ignore the following errors:
+extend-ignore =
+  E203,  # skipped because it annoys black
+  F401,  # skipped for unknown reason
+  WPS300,  # "Found local folder import" -- nothing bad about this
+  WPS306,  # "Found class without a base class: *" -- irrelevant under Python 3
```

Comment:
From the wemake docs:

Please, note that this rule has nothing to do with python2. We care only about consistency here, I like it here but I think the description for the exception could be better

There is a relate discussion about this one in the Discussions

Reply:
Ah, interesting. I just copied this from other projects where I disabled it because I don't agree with the sentiment. Consistency is good but I've never seen any of the upstream projects inheriting from `object` in every base class. As for Ansible projects, in core and collections `ansible-test` specifically has a `__metaclass__ = type` that makes py2 classes new-style enforcing this behavior in runtime.

Reply:
I have no issue with it being an exception, the explanation is misleading.....

Reply:
I can mention that a different kind of consistency is currently in place: https://github.com/ansible/ansible-navigator/discussions/700#discussioncomment-1791139.